kedro-airflow/README.md

```diff
@@ -56,6 +56,78 @@ You can use the additional command line argument `--jinja-file` (alias `-j`) to
 kedro airflow create --jinja-file=./custom/template.j2
 ```
 
+#### How can I pass arguments to the Airflow DAGs dynamically?
+
+`kedro-airflow` picks up configuration from `airflow.yml` files. The [parameters](https://docs.kedro.org/en/stable/configuration/parameters.html) are read by Kedro.
```

Comment:
Maybe add that this `airflow.yml` file should go in the `conf/base` or `conf/local` of your Kedro project. Or this could be in a folder starting with `airflow`